Runtime/Shaders/MapsBaker.cs

```diff
@@ -0,0 +1,217 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync {
+/// <summary>
+/// Helper class to bake smoothness and metallic into channels required for the standard shaders.
+/// </summary>
+internal static class MapsBaker {
+    private const string _SmoothnessTextureChannel = "_SmoothnessTextureChannel";
+
+    static Dictionary<string, ComputeShaderHelper> loadedShaders = new Dictionary<string, ComputeShaderHelper>();
+
+    private const string SHADER_CONST_METALLIC   = "Metallic";
+    private const string SHADER_CONST_SMOOTHNESS = "Smoothness";
+    private const string SHADER_CONST_RGB        = "RGB";
+
+    private const string SHADER_FILE                       = "meshsync_channel_mapping";
+    private const string SHADER_NAME_SMOOTHNESS_INTO_ALPHA = "smoothness_into_alpha";
+    private const string SHADER_NAME_HDRP_MASK             = "hdrp_mask";
+
+
+    private static ComputeShaderHelper LoadShader(string name) {
+        string file = SHADER_FILE;
+
+        if (!loadedShaders.TryGetValue(file, out var shaderHelper)) {
+#if UNITY_EDITOR
+            var shaderFiles = AssetDatabase.FindAssets(file);
+            if (shaderFiles.Length > 0) {
+                var shaderFile = shaderFiles[0];
+                var shader = AssetDatabase.LoadAssetAtPath<ComputeShader>(AssetDatabase.GUIDToAssetPath(shaderFile));
+
+                shaderHelper = new ComputeShaderHelper(shader, name);
+
+                loadedShaders[name] = shaderHelper;
+            }
+#else
+// TODO: Load shader at runtime here
```

Comment:
Can you convert this TODO into something like this, so that it's clear who to ask about this and when this todo was added ?
`[TODO-<person>:<date>] What to do`

Reply:
I've added a ticket number. We don't know who will do it yet.